tapir/shifts/views/management.py

```diff
@@ -251,10 +251,8 @@ def form_valid(self, form):
                     required_capabilities=entry.required_capabilities,
                     warnings=entry.warnings,
                 )
-        generate_shifts_up_to(
-            timezone.now().date()
-            + timezone.timedelta(days=len(form.cleaned_data["weekdays"]))
-        )
+                monday = get_monday(shift_template_copy_destination.start_date)
+                shift_template_copy_destination.group.create_shifts(monday)
```

Comment:
This will create new shifts for the entire group that the copy belongs to. We want to create shifts for `shift_template_copy_destination` exclusively.